docs/contributing/changes_from37to38.rst

```diff
@@ -0,0 +1,5 @@
+Changes from Python 3.7 to Python 3.8
+-------------------------------------
+
+.. literalinclude:: ast/python3_8.ast
+   :diff: ast/python3_7.ast
```

Comment:
This is a good use case for the grammar files. I did not even know this was possible.